### PR TITLE
fix: Prop types, unused props, replaced deprecated Main component

### PR DESCRIPTION
**Pull request by AsToNlele**
While working on inventory I was annoyed by the amount of prop type/other errors. Fixed some of them.

Some components don't use certain props so I excluded them from the spread.

`Main` component from `frontend-components` was deprecated for a while.

How to test:
Make sure the green section is the same like on stage
image

### CODE REVIEW

File: src/components/InventoryDetail/ApplicationDetails.js [hunk 1]
```diff
@@ -96,15 +96,26 @@ const ApplicationDetails = ({
...
-                />
-              ))}
+              {activeTabs?.map(
+                (
+                  {
+                    systemProfilePrefetched,
+                    component,
+                    nonEdge,
+                    tabRef,
+                    ...item
```
Comment (gkarat) [inline src/components/InventoryDetail/ApplicationDetails.js:106]: What's the goal of spreading the keys that are not used? For example, you explicitly extract `systemProfilePrefetched` or `component` key, but don't utilise it anywhere further down the code.

Comment (AsToNlele) [reply src/components/InventoryDetail/ApplicationDetails.js]: Nvm just removed the item spread, the props weren't used

File: src/routes/InventoryTable.js [hunk 2]
```diff
@@ -48,7 +47,7 @@ const Inventory = (props) => {
       <PageHeader className="pf-m-light">
         <PageHeaderTitle title="Systems" />
       </PageHeader>
-      <Main>
+      <section className="pf-v5-l-page__main-section pf-v5-c-page__main-section">
```
Comment (bastilian) [inline src/routes/InventoryTable.js:50]: Should/Could we maybe use the `PageSection` component here?

https://www.patternfly.org/components/page#pagesection

Comment (AsToNlele) [reply src/routes/InventoryTable.js]: yup, replaced it on more places

**Comment by AsToNlele** [comment]
Not sure that this error is fixable from Inventory, the `Tabs` component wants these props but the `button` at the bottom of the three doesn't.
image

File: src/components/InventoryDetail/ApplicationDetails.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ import {
   REPORTER_RHSM_CONDUIT,
   REPORTER_RHSM_PROFILE_BRIDGE,
 } from '../../Utilities/constants';
+import { PageSection } from '@patternfly/react-core/dist/esm';
```
Comment (gkarat) [inline src/components/InventoryDetail/ApplicationDetails.js:16]: nit-pick:
```suggestion
import { PageSection } from '@patternfly/react-core';
```

**Review by gkarat — APPROVED** [review]
LGTM!

**Pull request merged**